src/index.html

```diff
@@ -1,17 +1,133 @@
+              class="inputBlock"
+            >
+          </label>
+        </div>
+        <div>
+          <label>Name:
+            <input
+              type="text"
+              autocomplete="off"
+              class="inputBlock"
+            >
+          </label>
+        </div>
+        <div>
+          <label>How old are You?
+            <input
+              type="number"
+              min="1" max="100"
+              value="12"
+              class="inputBlock"
+            >
+          </label>
+        </div>
+        <div>
+          <label>Full date of birth:
+            <input type="date" class="inputBlock">
+          </label>
+        </div>
+        <div>
+          <label>I accept the term of the agreement
+            <input type="checkbox">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <div>
+          <label>E-mail:
+            <input
+              type="email"
+              placeholder="email@example.com"
+              class="inputBlock"
+            >
+          </label>
+        </div>
+        <div>
+          <label>Password:
+            <input type="password">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+        <div class="inputBlock">
+          <label >Do you love cats?
+            <input type="radio" name="Yes"> Yes
+            <input type="radio" name="No"> No
+          </label>
+        </div>
+        <div>
+          <label>What is your favorite color?
+            <input type="color" class="inputBlock">
+          </label>
+        </div>
+        <div>
+          <label>What time do you go to bed?
+            <input type="time" class="inputBlock">
+          </label>
+        </div>
+        <div class="inputBlock">
+          <label>What are your favorite brand of cars?
+            <select required multiple >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+              <option></option>
+            </select>
+          </label>
+        </div>
+        <div>
+          <label>How do you rate our work?
+            <input type="range">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Additional info:</legend>
+        <div class="inputBlock">
+          Comments:
+          <textarea minlength="2" maxlength="10"></textarea>
+        </div>
+        <div>
+          <label>Would you recommend us?
+            <select>
+              <option value="yes">yes</option>
+              <option value="no">no</option>
+            </select>
+          </label>
+        </div>
+      </fieldset>
+
+      <button type="submit">Submit</button>
+
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+
+</html>
```

Comment:
```suggestion
</html>

```
<img width="246" alt="Screenshot 2022-11-15 at 10 16 37" src="https://user-images.githubusercontent.com/94289038/201865928-38a7524d-9411-433f-98bf-4a9793a743ff.png">